tests/core/gui/test_workspace_button.py

```diff
@@ -273,6 +273,8 @@ async def test_workspace_button_delete_clicked(qtbot, workspace_fs, core_config,
 @pytest.mark.gui
 @pytest.mark.trio
 async def test_workspace_button_timestamped(qtbot, workspace_fs, core_config, alice_user_info):
+    if IS_OXIDIZED:
+        pytest.xfail("TODO: implement `.to_timestamped` method in rust")
```

Comment:
```suggestion
@pytest.mark.xfail(IS_OXIDIZED, reason="TODO: implement `.to_timestamped` method in rust")
async def test_workspace_button_timestamped(qtbot, workspace_fs, core_config, alice_user_info):
```
see [conditional parameter](docs.pytest.org/en/7.1.x/how-to/skipping.html#condition-parameter)

Reply:
e222d41bd71b49becac04b7170b75e4a8ac4a92b